### DISCUSSION TITLE
Organizations tab for admins

### DISCUSSION THREAD

**Discussion by lonix1** [woodpecker-ci/woodpecker#2257] [Request a Feature]
### Clear and concise description of the problem

The server config page shows tabs: "Secrets", "Users", "Agents" and "Queue".

There should be another tab, "Organizations".

There are various org-level settings, e.g. organization secrets (exposed by the API), but they aren't viewable in the UI. So they are effectively "hidden" and it's easy to forget about it.

### Suggested solution

A new tab named "Organizations".

Doesn't need to be complicated, or to allow interaction. Only to show the orgs, and secrets related to them. Readonly to keep it simple.

### Validations

- [X] Checked that the feature isn't part of the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by qwerty287** [comment]
For org secrets, you can just open the org page and use the wheel to open the secrets settings.

**Comment by 6543** [comment]
depend on #1873

**Comment by lonix1** [comment]
> ... just open the org page ...

There is an org page? :open_mouth:  :open_mouth: :open_mouth:

All I have is the (admin) user page, repositories and config. The admin user is an admin in gitea and member of an org.

(I'm using version "next")

**Comment by 6543** [comment]
well reread your isssue ... and yes your feature is already in the webUI ... nother the less we can use a org tab at the admin pannell from now on, as mentioned pull do store them

**Comment by lonix1** [comment]
How come I don't have an org page in the webUI? Sorry if this sounds silly, I'm surprised that I haven't seen it... Maybe I need my eyes checked! :)

**Comment by qwerty287** [comment]
Open a repo owned by the given org and click on the org's name in the top bar.

**Comment by qwerty287** [comment]
@anbraten Please, can we turn this back/reopen the issue and close the discussion?

There's nothing to discuss, everything's clear.
